Sources/Commands/SwiftPackageTool.swift

```diff
@@ -896,6 +899,226 @@ extension SwiftPackageTool {
         }
     }
 
+    struct InstallPackage: SwiftCommand {
```

Comment:
Let's put this type into a separate file so that this one stays as small as possible.

Reply:
Yup, think I'll close this specific PR, rewriting rn

Reply:
You may want to have a look at this PR which I opened before discovering your implementation https://github.com/apple/swift-package-manager/pull/6758. Please feel free to cherry-pick that commit into your rewrite branch if you'd like, I'll close my PR in favor of your implementation then since it's more complete with `update` and `remove` subcommands.